shardingsphere-infra/shardingsphere-infra-context/src/main/java/org/apache/shardingsphere/infra/context/refresher/type/DropViewStatementSchemaRefresher.java

```diff
@@ -39,8 +39,9 @@ public final class DropViewStatementSchemaRefresher implements MetaDataRefresher
     public Optional<MetaDataRefreshedEvent> refresh(final ShardingSphereDatabase database, final Collection<String> logicDataSourceNames,
                                                     final String schemaName, final DropViewStatement sqlStatement, final ConfigurationProperties props) throws SQLException {
         SchemaAlteredEvent event = new SchemaAlteredEvent(database.getName(), schemaName);
+        // TODO Drop view meta data from ShardingSphereSchema 's views
```

Comment:
Do you think `ShardingSphereSchema's views` is better?

Reply:
i will adjustment notes